docs/app/components/CheckboxGroupDocs/index.jsx

```diff
@@ -11,37 +11,161 @@ const CheckboxGroupDocs = () => (
   <DocsRoot
     title="CheckboxGroup"
     component={
-      <CheckboxGroup
-        name="checkbox-group[]"
-        heading="CheckboxGroup"
-        checkboxes={
-        [
-          {
+      <div>
+        <CheckboxGroup
+          name="checkbox-group[]"
+          heading="CheckboxGroup"
+          checkboxes={
+          [
+            {
+              id: 'option-a',
+              isChecked: true,
+              label: 'option 1',
+              value: 'option-1'
+            },
+            {
+              id: 'option-b',
+              isChecked: true,
+              label: 'option 2',
+              value: 'option-2'
+            },
```

Comment:
The docs should be matching the actual code of the component shown in the presentation section, so I guess we have to take out the last two objects?